agent-bootstrap/src/main/java/io/opentelemetry/auto/config/Config.java

```diff
@@ -173,14 +171,6 @@
 
     excludedClasses = getListSettingFromEnvironment(TRACE_CLASSES_EXCLUDE, null);
 
-    httpServerErrorStatuses =
-        getIntegerRangeSettingFromEnvironment(
-            HTTP_SERVER_ERROR_STATUSES, DEFAULT_HTTP_SERVER_ERROR_STATUSES);
-
-    httpClientErrorStatuses =
-        getIntegerRangeSettingFromEnvironment(
```

Comment:
`getIntegerRangeSettingFromEnvironment` is now used and can be removed?

Reply:
Nice catch lots of code removed